packages/components/pie-modal/README.md

```diff
@@ -52,6 +52,8 @@ import { PieModal } from '@justeattakeaway/pie-button/dist/react';
 | Property | Type      | Default | Description                                     |
 |----------|-----------|---------|-------------------------------------------------|
 | isOpen   | `Boolean` | `false` | Controls if the modal element is open or closed |
+| heading*   | `String` | - | Sets the heading of the modal |
```

Comment:
I think we should add in another column to state which props are required. Then you can mark the props you added as required `yes`

Reply:
I thought about the same thing but then decided to use `*` to indicate a prop is required as there will be few required props compared to optional and the table might be a bit hard to digest. Skip's [storybook](https://react.pie.design/?path=/story/components-modal--playground) has a similar table structure. 

as a side note, ideally, we should generate the metadata through jsDoc or the typescript definition as this approach will get out of sync easily but something to keep in mind

Reply:
Fully agree - we should automate this!